core/local/chokidar_event.js

```diff
@@ -2,12 +2,13 @@
 
 /*::
 import type fs from 'fs'
+import type {Metadata} from '../metadata'
 
 export type ChokidarAdd = {type: 'add', path: string, stats: fs.Stats}
 export type ChokidarAddDir = {type: 'addDir', path: string, stats: fs.Stats}
 export type ChokidarChange = {type: 'change', path: string, stats: fs.Stats}
-export type ChokidarUnlink = {type: 'unlink', path: string}
-export type ChokidarUnlinkDir = {type: 'unlinkDir', path: string}
+export type ChokidarUnlink = {type: 'unlink', path: string, old?: Metadata}
+export type ChokidarUnlinkDir = {type: 'unlinkDir', path: string, old?: Metadata}
```

Comment:
It's weird that you had to do the [exact same change](https://github.com/cozy-labs/cozy-desktop/pull/1275#discussion-diff-233448601L9) as @aenario in #1275

Reply:
Ok, got it.

Could you add a FIXME explaining that the offline unlink events detection is abusing the ChokidarUnlink* event types which actually never have an old property?

Reply:
@nono Forget about it, I'll fix it in a separate PR.

Reply:
What is really weird is that flow didn't tell us about that before